frame/vesting/src/lib.rs

```diff
@@ -182,24 +182,27 @@ pub mod module {
 	#[pallet::event]
 	#[pallet::generate_deposit(pub (crate) fn deposit_event)]
 	pub enum Event<T: Config> {
-		/// Added new vesting schedule. \[from, to, schedule\]
+		/// Added new vesting schedule.
 		VestingScheduleAdded {
 			from: AccountIdOf<T>,
 			to: AccountIdOf<T>,
 			asset: AssetIdOf<T>,
 			vesting_schedule_id: T::VestingScheduleId,
 			schedule: VestingScheduleOf<T>,
+			schedule_amount: BalanceOf<T>,
 		},
-		/// Claimed vesting. \[who, locked_amount\]
+		/// Claimed vesting.
 		Claimed {
 			who: AccountIdOf<T>,
 			asset: AssetIdOf<T>,
 			vesting_schedule_ids:
 				VestingScheduleIdSet<T::VestingScheduleId, T::MaxVestingSchedules>,
 			locked_amount: BalanceOf<T>,
 			claimed_amount: BalanceOf<T>,
+			amount_claimed_per_schedule:
```

Comment:
Nits: 
I think we can remove `claimed_amount` if we are adding this as it's redundant storage 
```suggestion
			claimed_amount_per_schedule:
```